README.md

```diff
@@ -31,10 +31,10 @@ SimuQ supports compilation to IonQ's trapped-ion devices and QuEra's neutral ato
 To enable these backends, you may install the dependencies via
 
 ```bash
-pip install simuq[braket]
+pip install "simuq[braket]"
 ```
 
-Then you need to configure AWS IAM roles, regions, and AWS credentials. Please refer to [Amazon Braket Python SDK](https://github.com/aws/amazon-braket-sdk-python).
+If running on QPUs, make sure that your AWS account is onboarded to Amazon Braket, as per the instructions [here](https://github.com/amazon-braket/amazon-braket-sdk-python#prerequisites) (this isn't necessary for running on the local simulator).
```

Comment:
The example notebooks below (for both Braket and IonQ providers) are broken, since they're moved to experimental (and were never well documented in the first place). I'd probably just replace with a generic link to the tutorial notebooks.